projects/core/src/lib/colors/color-palette-editor/color-palette-editor.component.html

```diff
@@ -0,0 +1,12 @@
+<button (click)="debugClick()">Print Debuginfo</button>
+<br />
+
```

Comment:
![grafik](https://user-images.githubusercontent.com/4947061/221539458-261a52d2-207a-43b7-9b68-b9774986b2b6.png)


If I select NDVI (classified), then switch to linear gradient and compute one, the linear gradient seems to work.
However, there are still 255 colors in the table, which seems wrong.

Reply:
Fixed this to display the color breakpoints of the linear gradient as palette values after applying the linear gradient. 142e1baeb77c0acfdc384be8ac8a2c5518d8f462